### DISCUSSION TITLE
Home routing when / is a redirect and menu highlight

### DISCUSSION THREAD

**Discussion by idc77** [quasarframework/quasar#10861] [General - Components / Directives / etc]
Quasar v2

routes.js
```js

const routes = [
  {
    path: '/',
    component: () => import('layouts/MainLayout.vue'),
    children: [
      { path: '', redirect: '/communities' },
      { path: 'communities', component: () => import('pages/CommunityList') },
      { path: 'communities/:id', component: () => import('pages/ContainerList') },
      { path: 'containers/:id', component: () => import('pages/ContainerDetail') },
      { path: 'forums/:id', component: () => import('pages/ForumDetail') },
      { path: 'threads/:id', component: () => import('pages/ThreadDetail') }
    ]
  },

  // Always leave this as last one,
  // but you can also remove it
  {
    path: '/:catchAll(.*)*',
    component: () => import('pages/Error404.vue')
  }
]

export default routes
```

I have a q-list with list items.
It's best demonstrated with images

While the home route is active, home is colored
![menuitem-athome](https://user-images.githubusercontent.com/87644834/135561930-5310807e-a90b-4fcf-b253-fe8753678552.png)

but when navigating to another route home remains colored
![menuitem-navigated](https://user-images.githubusercontent.com/87644834/135561949-d957c642-2aa5-4586-afc2-ed8ef56f72a3.png)

imho that's a bug

I can work around that by explicitly stating the redirected route `/communities`.
But the navigation display should be smart enough to resolve redirected routes and color the menu items accordingly

**Comment by MilosPaunovic** [comment]
Have you tried adding the `exact` prop on you `QItem`s?

**Reply by idc77** [reply]
I did, it has no effect

**Reply by seanjseymour** [reply]
For what it's worth, I had this exact same issue and adding the `exact` property to my q-items did fix it.
